From: Piotr Luc <piotr.luc@intel.com>
Subject: x86/intel_idle: Add Knights Mill CPUID
References: fate#321918
Patch-Mainline: v4.10-rc1
Git-commit: a2c1bc645e87346150516b3abf1933ed29d0f48b

Signed-off-by: Thomas Renninger <trenn@suse.de>

Add Knights Mill (KNM) to the list of CPUIDs supported by intel_idle.

Signed-off-by: Piotr Luc <piotr.luc@intel.com>
Reviewed-by: Dave Hansen <dave.hansen@intel.com>
Signed-off-by: Len Brown <len.brown@intel.com>

---
 drivers/idle/intel_idle.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/idle/intel_idle.c
+++ b/drivers/idle/intel_idle.c
@@ -1132,6 +1132,7 @@ static const struct x86_cpu_id intel_idl
 	ICPU(INTEL_FAM6_KABYLAKE_DESKTOP,	idle_cpu_skl),
 	ICPU(INTEL_FAM6_SKYLAKE_X,		idle_cpu_skx),
 	ICPU(INTEL_FAM6_XEON_PHI_KNL,		idle_cpu_knl),
+	ICPU(INTEL_FAM6_XEON_PHI_KNM,		idle_cpu_knl),
 	ICPU(INTEL_FAM6_ATOM_GOLDMONT,		idle_cpu_bxt),
 	ICPU(INTEL_FAM6_ATOM_DENVERTON,		idle_cpu_dnv),
 	{}
